### ISSUE TITLE
Apparently ext:geoip_IR.dat:ir Matches all addresses

### ISSUE THREAD

**Issue by PatrickstarWritesCode** [alireza0/x-ui#810]
I'm not sure if this is related to the panel source code or the upstream file providers, But all sites match with this rule, at least on my setup. If it's not related to the panel please kindly inform me to open an issue on https://github.com/Chocolate4U/Iran-v2ray-rules

Panel version: 1.6.4

```
[
  {
    "inboundTag": [
      "api"
    ],
    "outboundTag": "api",
    "type": "field"
  },
  {
    "ip": [
      "geoip:private"
    ],
    "outboundTag": "blocked",
    "type": "field"
  },
  {
    "type": "field",
    "port": "53",
    "network": "tcp,udp",
    "outboundTag": "direct"
  },
  {
    "type": "field",
    "outboundTag": "blocked",
    "protocol": [
      "bittorrent"
    ]
  },
  {
    "type": "field",
    "outboundTag": "warp",
    "domain": [
      "domain:openai.com",
      "domain:ai.com",
      "geosite:google",
      "domain:spotify.com",
      "domain:ir",
      "geosite:category-ir",
      "ext:geoip_IR.dat:ir"
    ]
  }
]
```

**Comment by Enkidu-6** [comment]
Yes I noticed that a while ago. It implies to have Iranian IP addresses listed but it acts like a catch all. I removed all references to that database from my rules and I only use the original geoip.dat until I know exactly what's included in those files. It does the same if it ends with :arvancloud or :derakcloud , etc ...

For itanian sites I use:
```
         "regexp:.*\\.ir$",
          "regexp:.*\\.xn--mgba3a4f16a$",
          "geosite:category-ir",
          "geoip:ir",
          "ext:geosite_IR.dat:ir"

```
I haven't confirmed the accuracy of  ***"ext:geosite_IR.dat:ir"***  either but it's supposed to contain Iranian domains that have a non-ir TLD. For now I'm using it because I haven't seen it cause any problems.

As for the rest of your rules, openai and spotify actually have their own categories so I'd replace those rules with:

```
"geosite:openai",
"geosite:spotify"
```
Cheers
